packages/react/src/executors/module-federation-dev-server/module-federation-dev-server.impl.ts

```diff
@@ -38,9 +41,24 @@ export default async function* moduleFederationDevServer(
     );
   }
 
+  const patternMatchedRemotesToSkip = new Set(
+    findMatchingProjects(options.skipRemotes, context.projectGraph.nodes)
+  );
+  const remotesNotInGraph = options.skipRemotes.filter((r) => {
+    const { exclude, type } = parseStringPattern(r, context.projectGraph.nodes);
+    return exclude === false && type === 'unlabeled';
+  });
   const remotesToSkip = new Set(
-    findMatchingProjects(options.skipRemotes ?? [], context.projectGraph.nodes)
+    [...patternMatchedRemotesToSkip, ...remotesNotInGraph] ?? []
   );
+
+  if (remotesToSkip.size > 0) {
+    console.log(
```

Comment:
Let's use the Nx `logger`:
```suggestion
    logger.info(
```